src/modules/showScores.js

```diff
@@ -0,0 +1,18 @@
+import Scores from './addScore.js';
+import rowDOM from './addDOMelement.js';
+
+const refresh = () => {
+  const buttonRefresh = document.querySelector('.table-container__button');
+  buttonRefresh.addEventListener('click', async () => {
+    const table = document.querySelector('.table-container__body');
+    // Resets inner HTML so that data won't be repeated
+
+    table.innerHTML = '';
+    const data = new Scores();
+    // Got array of data from API: scores and names.
+    const scoresAPI = await data.getScores();
+    rowDOM(table, scoresAPI);
+  });
```

Comment:
- [ ] One of the key requirements for this milestone is that you follow the wireframe even though styling is not important. Please try to display the leaderboard scores correctly while following the wireframe design. Attached below is your current display 👇 
### Wireframe
![image](https://github.com/EvansSnave/leaderboard/assets/46839548/8fe16ea4-1112-4f13-9404-925742ba38e8)

### Your Implementation
![image](https://github.com/EvansSnave/leaderboard/assets/46839548/62946220-fe9c-4643-9277-5ef5a6d46239)